counter_examples.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+make clean && make
+
+make run &
+
+# wait until program is ready to listen
+# not sure how else to it now
+sleep 3;
```

Comment:
This is a common problem on UNIX, modern distribution have SD notify API by systemd. However for a test this is probably fine and it is not worth implementing a special handling of this. At least for now, tests will look different after the code base is integrated into PCP.